### ISSUE TITLE
Feature request: Add another Polkassembly endpoint

### ISSUE THREAD

**Issue by wischli** [polkadot-js/apps#6124]
Hello, I was wondering whether it is possible if we can add another Polkassembly endpoint (*.network instead of *.io). [This is a fork by Premiurly](https://github.com/Premiurly/polkassembly) which supports KILT Protocol, Moonriver, Karura, Khala among others.

If so, should this be part of _packages/apps-config/src/links/polkassembly.ts_ or moved to a separate file?

### Draft
```diff
export default {
  chains: {
-    Kusama: 'kusama',
-     'Kusama CC3': 'kusama',
-     Polkadot: 'polkadot'
+    'KILT Peregrine': 'kilt',
+    Karura: 'karura',
+    Khala: 'khala',
+    Moonriver: 'moonriver'
  },
  create: (chain: string, path: string, data: BN | number | string): string =>
- `https://${chain}.polkassembly.io/${path}/${data.toString()}`,
+ `https://${chain}.polkassembly.network/${path}/${data.toString()}`,
  isActive: true,
  logo: externalLogos.polkassembly as string,
  paths: {
    bounty: 'bounty',
    council: 'motion',
    proposal: 'proposal',
    referendum: 'referendum',
    tip: 'tip',
    treasury: 'treasury'
  },
- url: 'https://polkassembly.io/'
+ url: 'https://polkassembly.network/'
};
```

cc @niklabh

**Comment by jacogr** [comment]
Sure. As Long as the fork is approved by the owner (which is this case seems like it is), no issues adding another endpoint. They should be simple to get going.

**Comment by wischli** [comment]
Should this be added as a custom export to the same _polkassembly.ts_ file or moved to another one like _polkassemblyNetwork.ts_?

**Comment by jacogr** [comment]
Either way. I generally split, but since most are shared can just create a shared object and have only the differences changed 

TL;DR would be happy with both
